cloud/lxd/lxd_container.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Hiroaki Nakamura <hnakamur@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = """
+---
+module: lxd_container
+short_description: Manage LXD Containers
+version_added: 2.2.0
+description:
+  - Management of LXD containers
+author: "Hiroaki Nakamura (@hnakamur)"
+options:
+    name:
+        description:
+          - Name of a container.
+        required: true
+    config:
+        description:
+          - a config dictionary for creating a container.
+            See https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1
+          - required when the container is not created yet and the state is
+            not absent.
+        required: false
+    state:
+        choices:
+          - started
+          - stopped
+          - restarted
+          - absent
+          - frozen
+        description:
+          - Define the state of a container.
+        required: false
+        default: started
+    timeout_for_addresses:
+        description:
+          - a timeout of waiting for IPv4 addresses are set to the all network
+            interfaces in the container after starting or restarting.
+          - if this values is equal to or less than 0, ansible does not
+            wait for IPv4 addresses.
+        required: false
+        default: 0
+requirements:
+  - 'lxd >= 2.0 # OS package'
```

Comment:
Does the `# OS package` add any extra information? I think it could confuse the reader

Does it matter if lxd (or python, or pylxd) are installed via apt/yum or pip?